client_lib/pulp/client/commands/repo/query.py

```diff
@@ -34,7 +34,7 @@ def run(self, **kwargs):
         # Limit to only repositories of a specific type
         if kwargs.get('str-eq', None) is None:
             kwargs['str-eq'] = []
-        kwargs['str-eq'].append(['notes.%s' % constants.REPO_NOTE_TYPE_KEY, self.repo_type])
+        kwargs['str-eq'].append(['notes__%s' % constants.REPO_NOTE_TYPE_KEY, self.repo_type])
```

Comment:
This should be fixed on the server side using the field mapping stuff we worked on earlier today.